components/create_post/create_post.jsx

```diff
@@ -1123,16 +1125,21 @@ class CreatePost extends React.PureComponent {
         this.setState({renderScrollbar: height > maxHeight});
     }
 
+    readOnlyChannel = () => {
```

Comment:
Total nitpicking, but maybe call this `isReadOnlyChannel`? I personally prefer getters to be called `isX` when it returns a boolean